### PR TITLE
[4.0] Remove unused strings

### PR DESCRIPTION
**Pull request by brianteeman**
Having checked the code I can't see that there is any circumstance where these strings can ever be used either when checking in an item or setting a home menu

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_menus.ini [hunk 1]
```diff
@@ -113,9 +113,7 @@ COM_MENUS_ITEMS_REBUILD_FAILED="Failed rebuilding Menu Items list."
 COM_MENUS_ITEMS_REBUILD_SUCCESS="Menu items list rebuilt."
 COM_MENUS_ITEMS_SEARCH_FILTER="Search in title, alias and notes. Prefix with ID: to search for a menu item ID."
 COM_MENUS_ITEMS_SEARCH_FILTER_LABEL="Search Menu Items"
-COM_MENUS_ITEMS_SET_HOME_0="No menu item set to home."
 COM_MENUS_ITEMS_SET_HOME_1="1 menu item set to home."
-COM_MENUS_ITEMS_SET_HOME_MORE="%d menu items set to home."
```
Comment (infograf768) [inline administrator/language/en-GB/en-GB.com_menus.ini]: This one has to stay as we can set to Home multiple menu items when in multilingual.
screen shot 2018-06-16 at 10 14 07

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.com_menus.ini]: Done

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 82c384ede16d98daed04203dfaa453d0e268a17c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20752">issues.joomla.org/tracker/joomla-cms/20752</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 82c384ede16d98daed04203dfaa453d0e268a17c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20752">issues.joomla.org/tracker/joomla-cms/20752</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20752">issues.joomla.org/tracker/joomla-cms/20752</a>.</sub>

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**